### PR TITLE
Add links in internal errors

### PR DESCRIPTION
**Pull request by lionel-**

### CODE REVIEW

File: R/cnd-abort.R [hunk 1]
```diff
@@ -665,17 +665,29 @@ utils::globalVariables(".internal")
...
 footer_internal <- function(env) {
   top <- topenv(env)
+  url_line <- NULL
+
   if (is_namespace(top)) {
-    pkg <- sprintf(" in the %s package", ns_env_name(top))
+    pkg <- ns_env_name(top)
+    url <- pkg_url_bug(pkg)
+    pkg_line <- sprintf(
+      "This is an internal error that was detected in the %s package.",
```
Comment (DavisVaughan) [inline R/cnd-abort.R:674]: I find it somewhat funny that we have all these nice cli-like helpers for formatting but still use `sprintf()` for interpolation

File: R/cnd-abort.R [hunk 2]
```diff
@@ -665,17 +665,29 @@ utils::globalVariables(".internal")
...
+      format_pkg(pkg)
+    )
+
+    url <- pkg_url_bug(pkg)
```
Comment (DavisVaughan) [inline R/cnd-abort.R:678]: You also do this on line 672

File: R/cnd-abort.R [hunk 3]
```diff
@@ -665,17 +665,29 @@ utils::globalVariables(".internal")
...
+    if (!is_null(url)) {
+      url_line <- sprintf(
+        "Please report it at %s with a %s and the full backtrace.",
+        format_url(url),
+        format_href("reprex", "https://reprex.tidyverse.org/")
```
Comment (DavisVaughan) [inline R/cnd-abort.R:683]:
```suggestion
        format_href("reprex", "https://tidyverse.org/help")
```

I actually send people to this link instead https://tidyverse.org/help, I find it to be the clearest example of what we actually want them to do - up to you about which link to use

**Review by DavisVaughan — APPROVED** [review]
I love it!

File: R/cnd-abort.R (hunk 3 above)
Comment (lionel-) [reply R/cnd-abort.R:683]: oh perfect

**Pull request merged**